doc/api/timers.markdown

```diff
@@ -16,6 +16,8 @@ It is important to note that your callback will probably not be called in exactl
 the callback will fire, nor of the ordering things will fire in. The callback will
 be called as close as possible to the time specified.
 
+To follow browser behavior, when using delays larger than 2147483647 milliseconds (approximately 25 days), the timeout is executed immediately, as if the `delay` was set to 1.
```

Comment:
Maybe say "If `delay` is less than one, or greater than the maximum allowed value of 2147483647, it will be assigned a value of 1."